XenAdmin/TabPages/GeneralTabPage.cs

```diff
@@ -1329,6 +1346,16 @@ private void generateGeneralBox()
             s.AddEntry(FriendlyName("host.uuid"), GetUUID(xenObject));
         }
 
+        private string PoolExpireString()

```

Comment:
Rename this to "PoolAdditionalLicenseString" (or something similar), as it returns additional information about the license (unlicensed or expired)